bin/asinit.js

```diff
@@ -427,13 +427,16 @@ function ensureTestsDirectory() {
 }
 
 function ensureTestsIndexJs() {
-  console.log("- Making sure that 'tests/index.js' exists...");
+  console.log("- Making sure that 'tests/index.test.js' exists...");
```

Comment:
Maybe we can have the directory be `test/` instead of `tests/`, so the file path can be `test/index.js`. This would let the script command in `package.json` be `node --test`, because [by](https://nodejs.org/docs/latest-v20.x/api/test.html#test-runner-execution-model) [default](https://nodejs.org/api/test.html#running-tests-from-the-command-line:~:text=**/test/,js}) it includes anything under the `test/` directory.